database/programs.json

```diff
@@ -1166,7 +1177,26 @@
       "f2e9c480af31a4039af02dd7a2b8d5d1f859704d": {
         "file": "ZeroPong [zeroZshadow, 2007].ch8",
         "platforms": ["originalChip8"]
+      },
+      "f26993a4afd5cda2fea19935773fd3db54866623": {
```

Comment:
Interesting, did the CHIP-8 Archive update since the last PR? A lot of these titles look like they should already be in the DB. I'll look into this.

Reply:
I'm wondering if the hash function in [update-chip8-archive.js](https://github.com/chip-8/chip-8-database/blob/master/scripts/update-chip8-archive.js) was changed.